### ISSUE TITLE
tiny url's

### ISSUE THREAD

**Issue by UseDotCom** [UseDotCom/lighthouse#704]
Created by **[usedotcom](https://github.com/usedotcom)**  at 12:44Z on 03/25/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

We need to create tiny/shortened url's for sharing. This would apply to individual photo url's, photo set url's, supersize page url's, and (maybe) Saved Template url's. Let's discuss what this entails and a time frame.

- - - -

Lighthouse ticket URL: http://use-com.lighthouseapp.com/projects/103799/tickets/286

Attachments:

* https://use-com.lighthouseapp.com/attachments/4956088/sharedrop.png
* https://use-com.lighthouseapp.com/attachments/4956089/gencode.png
* https://use-com.lighthouseapp.com/attachments/4956942/Screen_Shot.png
* https://use-com.lighthouseapp.com/attachments/4994985/viewer-issues.png
* https://use-com.lighthouseapp.com/attachments/5017951/viewer-codes-tiny-urls.png

Was assigned to **[](https://github.com/)**

**Comment by UseDotCom** [comment]
By **Dave** at 02:48Z on 04/01/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

1) Encoding/decoding of URLs working
2) Should have share links finished tomorrow.

**Comment by UseDotCom** [comment]
By **Dave** at 03:04Z on 04/07/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

I'm still making sure I have all of the share links, but if I do, I'm done.

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 17:02Z on 04/07/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

I'm assuming this isn't anywhere I can test, as we're waiting on deployment.

**Comment by UseDotCom** [comment]
By **Dave** at 17:50Z on 04/07/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

That's correct.

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 13:27Z on 05/01/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

I found one so far that doesn't create a proper tiny url. Other videos have worked properly. This is in an admin account 

http://arabella7.desktopcurrency.com/gencodes.pl?set=9e672485ebfa1ad53b5a

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 13:49Z on 05/01/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

The Blog URL is not generating tiny url's properly. This occurs in the Generate HTML code box pop up and the Share dropdown

http://lafferty.desktopcurrency.com/gencodes.pl?set=d3ccf9d63bb59f262b26

http://lafferty.desktopcurrency.com/gencodes.pl?set=74bbffdd569a94a32cfc

**Comment by UseDotCom** [comment]
By **Ura** at 16:59Z on 05/01/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

The pop up code box after uploading a video, tiny url didnt work and sent me to something else.

**Comment by UseDotCom** [comment]
By **Dave** at 03:50Z on 05/05/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

Fixed. I had to reset the table to make this work, so any existing short links will not work (though new ones will automatically be generated).

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 19:21Z on 05/12/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

If these are not directly related to the tiny url ticket, you can move them into their own tickets.

1)The share links on instances of the gencodes pop up (for example, click Generate HTML & Image links on any photo set in your account) are only displaying www.desktopcurrency.com, not the set's url or tiny url.

2)None of the share icons work on newid pop up (post-upload) 

3)HTML codes on viewer pages do not incorporate the tiny url (see attached)

**Comment by UseDotCom** [comment]
By **Dave** at 04:50Z on 05/14/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

1 is fixed. Working on 2.

**Comment by UseDotCom** [comment]
By **Dave** at 04:07Z on 05/15/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

2 & 3 fixed. Was this all we had for remaining issues?

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 15:24Z on 05/15/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

1) The share bar on gencodes is now displaying http://www.desktopcurrency.com/sQ for all sets. Checked multiple sets and multiple users.

2 & 3 are unchanged.

**Comment by UseDotCom** [comment]
By **Dave** at 17:07Z on 05/15/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

2 & 3 (partially) are working for me, unless I'm looking in the wrong places.

I did find there's a bug where 3 won't work if you're not logged in and I'm working on a fix. What are your steps for #2? Are you logged in? What upload page are you using?

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 17:39Z on 05/15/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

2) Upload images to your account. You receive the neweid code popup. Clicking the Share icons has no effect.

3) This is not accessed via your editset (owner). Go to Most Recent. Click on a thumbnail to open the viewer page. Click on Show Links & HTMl codes. It does not use the tiny url's. There is a screenshot attached on my original post about this.

**Comment by UseDotCom** [comment]
By **Dave** at 02:30Z on 05/16/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

#2 fixed.

#3 was intertwined with the issue I described above, fixed.

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 15:52Z on 05/16/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

2) Fixed

1) Not fixed
3) Not fixed. I attached a fresh screenshot.

**Comment by UseDotCom** [comment]
By **Dave** at 03:32Z on 05/19/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

1) Fixed

3) API nearly finished. Should have this sometime tomorrow.

**Comment by UseDotCom** [comment]
By **Dave** at 00:16Z on 05/20/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

#3 is done.

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 17:26Z on 06/24/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

Copied over from the Supersize ticket.

The thumbnail in the share via email shows as moved or deleted.

A sample url given is http://www.desktopcurrency.com/images/s_1/BiJ.jpg

This seems to occur in all email share instances since the implementation of the tiny urls.

**Comment by UseDotCom** [comment]
By **Dave** at 20:52Z on 06/24/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

I just put a fix in. Try now.

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 13:48Z on 06/25/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

Working except in Profile email shares. Those instances are being moved to a separate ticket #337 and will be addressed post 6/25 deployment

**Comment by UseDotCom** [comment]
By **James** at 14:28Z on 06/25/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/286))*

- - - -

Tiny URLs seem to be working properly.

**Comment by UseDotCom** [comment]
Imported from Lighthouse  at 21:06Z on 02/05/2015

Assigned to ticket creator, **[usedotcom](https://github.com/usedotcom)**
